README.md

```diff
@@ -3,30 +3,26 @@ Setup an SSH socket with a private key.
 
 ## Usage
 ### Inputs
-#### `host`
-**Required** Remote hostname
 
-#### `port`
-**Optional** SSH port
+Following inputs can be used as `step.with` keys
 
-This is only necessary if the SSH server does not listen on port 22. It is used to retrieve the host key so it can be trusted for host key verification.
+| Name             | Required | Type    | Description                        |
+|------------------|----------|---------|------------------------------------|
+| `host` | Yes | String | Remote hostname. |
+| `port` | No | Number | SSH Port (default: `22`). |
+| `socket-path` | Yes | String | Path at which to create socket. |
+| `key` | Yes | String | base64 private key |
 
-#### `socket-path`
-**Required** Path at which to create socket.
-
-#### `key`
-**Required** SSH private key as base64
-
-Base64 encode your key:
+You can generate a base64 iteration of the private key via:
```

Comment:
idk if iteration is the right word here.

Reply:
Reworded. Let me know